public/student.html

```diff
@@ -331,11 +334,11 @@ <h1 class="hyphenate" id="alina">Alina Spera <span style="font-size: 1rem;">(she
       </div>
       <div class="col-lg-6  text-left  pl-lg-5 pl-1 pt-lg-0 pt-2 py-5 d-flex flex-column justify-content-lg-around">
          <div class>
-            <h1 id="sydney">Sydney Wilkinson</h1>
+            <h3 class="h1" id="sydney">Sydney Wilkinson</h3>
             <p class="lead border-dark border-bottom">University of Alaska Fairbanks, College of Fisheries and Ocean Sciences</p>
             <p>My doctoral work focuses on aquatic systems and trophic linkages that shape food web form and function. I am interested in determining what energy sources (e.g., terrestrial, bacterial, phytoplanktonic) fuel food webs and how these sources change through time and space. I use biomarkers (e.g., stable isotopes, fatty acids) as tools to better understand trophic connectedness and resilience of food webs. It is important to me to work in environments experiencing rapid change to better understand the implications of climate change at a global scale.</p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:swilkinson4@alaska.edu"><i class="  far fa-paper-plane"></i>  swilkinson4@alaska.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0002-5244-7213" target="_blank">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://orcid.org/0000-0002-5244-7213" target="_blank" aria-label="open page in new tab">Publications</a></button>
```

Comment:
specify what you're opening in new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5